xdem/coreg.py

```diff
@@ -1979,7 +1997,14 @@ def subdivide_array(self, shape: tuple[int, ...]) -> NDArrayf:
             shape = (shape[1], shape[2])
         return spatial_tools.subdivide_array(shape, count=self.subdivision)
 
-    def _apply_func(self, dem: NDArrayf, transform: rio.transform.Affine, **kwargs: Any) -> NDArrayf:
+    def _apply_func(
+        self, dem: NDArrayf, transform: rio.transform.Affine, crs: rio.crs.CRS, **kwargs: Any
+    ) -> tuple[NDArrayf, rio.transform.Affine]:
+
+        # The option resample=False is not implemented for this case
+        if "resample" in list(kwargs.keys()):
```

Comment:
Is this not as simple as `"resample" in kwargs and kwargs["resample"] is not True`?

Note the more explicit boolean comparison. `not None` or `not ""` is also True.

`is False` is of course easier, but then a weirdly typed resample would be okay, like `resample = None`

Reply:
Yes, seems to be working too, thanks!
Changed in bde8967